src/object.c

```diff
@@ -233,7 +233,7 @@ robj *dupStringObject(const robj *o) {
 }
 
 robj *createQuicklistObject(void) {
-    quicklist *l = quicklistCreate();
+    struct quicklist *l = quicklist_new(server.list_max_listpack_size, server.list_compress_depth);
```

Comment:
These look like very unnecessary changes.

Please revert all renamings of functions (also quicklistRelease and others) to minimize the diff. The key to have the PR accepted is to make it small.

Reply:
Fewer statements, fewer bugs. See issue #12871.

Reply:
we know this, but this PR has changed a lot, and we don’t have enough time or energy to review it in the short time, so we would like to ask you to try to ensure that the diff is as small as possible.

Reply:
I've talked about this at the beginning of the commit log. I've tried to fix them one by one and failed.

Reply:
https://github.com/redis/redis/issues/12864#issuecomment-1857192020

can you respond this so that we can came up a way to migrate the diff?